cips/cip-14.md

```diff
@@ -213,7 +213,18 @@ This proposal is backwards-incompatible because it is state-machine breaking. Th
 
 ## Test Cases
 
-TBA
+The following test cases are available in the [ibc-go e2e repository.](https://github.com/cosmos/ibc-go/tree/main/e2e/tests/interchain_accounts)
+
+- Registration of an interchain account
+- Transfer funds from interchain account to a different account on the same chain using an unordered or ordered channel
+- A failed transfer of funds from interchain account to a different account on the same chain due to insufficient funds
```

Comment:
Can this test clarify if "insufficient funds" applies to the gas fees for the transfer or if the transfer amount exceeds the balance of the sending account